### PR TITLE
ssh-keygen: add -R example

### PR DESCRIPTION
**Pull request by half2me**
Add `ssh-keygen -R` command

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ssh-keygen.md [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 `ssh-keygen -l -F {{remote_host}}`
 
+- Remove the keys of a host from your known_hosts file (useful when a known host has a new key):
+
+`ssh-keygen -R {{myhost.com}}`
```
Comment (agnivade) [inline pages/common/ssh-keygen.md]: s/myhost.com/remote_host

Comment (half2me) [reply pages/common/ssh-keygen.md]: ah, I thought I had already changed that :)

File: pages/common/ssh-keygen.md [hunk 2]
```diff
@@ -22,6 +22,10 @@
 
 `ssh-keygen -l -F {{remote_host}}`
 
+- Remove the keys of a host from your known_hosts file (useful when a known host has a new key):
```
Comment (agnivade) [inline pages/common/ssh-keygen.md]: Sorry I missed it in the first pass, but we typically avoid mentioning the 2nd person in our pages. But I also see that the 4th example also has "your". Would be open to sending another change replacing "your" with the/one's ? It is completely fine if you don't want to. :)

Comment (half2me) [reply pages/common/ssh-keygen.md]: sure

Comment (half2me) [reply pages/common/ssh-keygen.md]: I've changed both instances

**Pull request merged**